libnetdata/ebpf/ebpf.c

```diff
@@ -0,0 +1,99 @@
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+
+#include "../libnetdata.h"
+
+static int clean_kprobe_event(FILE *out, char *filename, char *father_pid, netdata_ebpf_events_t *ptr) {
+    int fd =  open(filename, O_WRONLY | O_APPEND, 0);
+    if (fd < 0) {
+        if(out) {
+            fprintf(out, "Cannot open %s : %s\n", filename, strerror(errno));
+        }
+        return 1;
+    }
+
+    char cmd[1024];
+    int length = sprintf(cmd, "-:kprobes/%c_netdata_%s_%s", ptr->type, ptr->name, father_pid);
+    int ret = 0;
+    if (length > 0) {
+        ssize_t written = write(fd, cmd, strlen(cmd));
+        if (written < 0) {
+            if(out) {
+                fprintf(out
+                        , "Cannot remove the event (%d, %d) '%s' from %s : %s\n"
+                        , getppid(), getpid(), cmd, filename, strerror((int)errno));
+            }
+            ret = 1;
+        }
+    }
+
+    close(fd);
+
+    return ret;
+}
+
+int clean_kprobe_events(FILE *out, int pid, netdata_ebpf_events_t *ptr) {
+    //
+    debug(D_EXIT, "R.I.P. dad I will clean your events.");
```

Comment:
🤔 ?

Reply:
I confess I was without ideas to write something, but I needed to have some sentence to debug. This message is in this place, because the forked child assumed the controller becoming an independent process running, and it will clean everything that the parent process left. 
Without to have a fork I cannot clean the eBPF programs loaded.
I accept any suggestion to improve this sentence! ;)

Reply:
i stressed it because it looks like one of those `KILL ME` messages you use during developing.

Reply:
I understood, I am removing it now.

Reply:
I updated the message for something more meaninful